library/src/main/java/org/xmtp/android/library/libxmtp/XMTPLogger.kt

```diff
@@ -0,0 +1,10 @@
+package org.xmtp.android.library.libxmtp
+
+import android.util.Log
+import uniffi.xmtpv3.FfiLogger
+
+class XMTPLogger : FfiLogger {
+    override fun log(level: UInt, levelLabel: String, message: String) {
+        Log.i("$level $levelLabel", message)
```

Comment:
Would be nice to actually log at different levels by transforming the Rust log levels to the Android log levels.